pkg/trivy/wrapper.go

```diff
@@ -121,6 +121,11 @@ func (w *wrapper) prepareScanCmd(imageRef ImageRef, outputFile string) (*exec.Cm
 		args = append([]string{"--skip-update"}, args...)
 	}
 
+	if w.config.IgnorePolicy != "" {
+		args = append([]string{"--ignore-policy",w.config.IgnorePolicy}, args...)
```

Comment:
Use gofmt to fix formatting

Reply:
Hi @danielpacak 
I've done your requested changed to the PR.